### ISSUE TITLE
Podspec is failing

### ISSUE THREAD

**Issue by Dids** [DevMate/DevMateKit#1]
```ruby
Downloading dependencies
Installing DevMateKit (1.1.1)
[!] /bin/bash -c 
set -e
RUBY_SCRIPT='copy_xpc_build_phase.rb'; if [ -f $RUBY_SCRIPT ]; then ruby $RUBY_SCRIPT 'DevMateKit.podspec'; fi

Current directory: /Users/Dids/Library/Caches/CocoaPods/Pods/Release/DevMateKit/1.1.1-35e88

Path to spec: DevMateKit.podspec

Installing from spec repo...
copy_xpc_build_phase.rb:27:in `+': no implicit conversion of nil into String (TypeError)

	from copy_xpc_build_phase.rb:27:in `<main>'
```

**Comment by Dids** [comment]
False alarm, it was probably a temporary network issue as it's working fine now. :)

**Comment by Dids** [comment]
Getting this again with version 1.1.3, no idea why. Using the latest version of Cocoapods.

**Comment by Dids** [comment]
Just ran it with --verbose and got the following:

```ruby
 > Running prepare command
   $ /bin/bash -c  set -e RUBY_SCRIPT='copy_xpc_build_phase.rb'; if [ -f
   $RUBY_SCRIPT ]; then ruby $RUBY_SCRIPT 'DevMateKit.podspec'; fi
   copy_xpc_build_phase.rb:27:in `+': no implicit conversion of nil into String (TypeError)
   	from copy_xpc_build_phase.rb:27:in `<main>'
   Current directory: /Users/Dids/Library/Caches/CocoaPods/Pods/Release/DevMateKit/1.1.3-89ba5
   Path to spec: DevMateKit.podspec
   Installing from spec repo...
<snip>
copy_xpc_build_phase.rb:27:in `+': no implicit conversion of nil into String (TypeError)
	from copy_xpc_build_phase.rb:27:in `<main>'
```

Could it be a path issue with the ruby script?

**Comment by dimaty** [comment]
Yes it is. But according to your log i can't understand why ruby script runs from Caches folder. Can you remove Caches folder and update pod again? You can use script https://gist.github.com/mbinna/4202236 for this.

**Comment by Dids** [comment]
Same error I'm afraid. I'm using the latest version of both OS X Yosemite and Xcode.

Ruby version: ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
Cocoapods version: 0.37.2

**Comment by dimaty** [comment]
Ok. I've found steps that lead to the problem. It will be fixed in the near future.

**Comment by Dids** [comment]
Thanks! I'll link the framework manually in the meantime.

**Comment by dimaty** [comment]
Hi there.
We've fixed this issue with new release v1.2. Could you, please, check it and inform us if it works? Before updating pods you need to update your Podfile with next lines
````
platform :osx, '10.7'
pod 'DevMateKit'

post_install do |installer|
    system("RUBY_SCRIPT='Pods/DevMateKit/copy_xpc_build_phase.rb'; if [ -f $RUBY_SCRIPT ]; then ruby $RUBY_SCRIPT '#{path}'; fi")
end
````

**Comment by Dids** [comment]
Seems to be working fine now, even without the post_install script part. I'm on a different Mac now though, so not sure if the issue exists on this specific Mac.
I'll still close the issue though, thank you!
